addon/controllers/project/form.js

```diff
@@ -84,7 +85,14 @@ export default class ProjectFormController extends Controller {
         this.intl.t("ember-gwr.constructionProject.saveSuccess")
       );
     } catch (error) {
-      console.error(error);
+      const errors = JSON.parse(error.message);
+      this.errors = [
+        ...(errors.error.length || !errors.errorList.length
+          ? [this.intl.t("ember-gwr.generalErrors.genericFormError")]
+          : []),
+        ...errors.errorList.map((error) => error.messageOfError),
+      ];
+      console.log(error); // eslint-disable-line no-console
```

Comment:
With `error` you dont need the `eslint-disable`.
```suggestion
      console.error(error);
```